.github/workflows/release.yml

```diff
@@ -119,8 +119,8 @@ jobs:
 
   slack-notify:
     needs:
-      - "publish_gh"
-      - "publish_pypi"
+      - "publish_python"
+      - "container-publish"
```

Comment:
Can we change these two names to be consistent while we're in here? `publish_python` and `publish_docker` perhaps?